test/test_utils/__init__.py

```diff
@@ -1270,6 +1270,25 @@ def get_framework_and_version_from_tag(image_uri):
             "1.8.0": "1.8.0.2.2.2.0",
         },
     },
+    "1.19.1": {
```

Comment:
@aws-vrnatham suggested adding this to resolve the sanity test failure, which seems to exist as a result of 1.19.1 not being referenced here